ext/standard/array.c

```diff
@@ -4192,6 +4193,7 @@ PHP_FUNCTION(array_column)
 
 	if ((column && !array_column_param_helper(column, "column")) ||
 	    (index && !array_column_param_helper(index, "index"))) {
+		/* Throw exception ? */
```

Comment:
this is not a warning / type-error, so I think it shouldn't be converted to an exception as part of this PR.
I think you should remove this comment from this PR

Reply:
All cases where array_column_param_helper returns false will already throw an exception after this change, so this branch should just be changed to `return;` to clarify that no value will actually be returned.